docs/architecture/dapr-for-net-developers/getting-started.md

```diff
@@ -354,90 +335,90 @@ In the final part of this example, you'll add container support and run the solu
 
-    The *Dockerfile* contains the YAML:
+    The *Dockerfile* contains the following commands:
 
-    ```yml
-    FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
+    ```dockerfile
+    FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
     WORKDIR /app
     EXPOSE 80
     EXPOSE 443
 
-    FROM mcr.microsoft.com/dotnet/sdk:5.0 AS build
+    FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
     WORKDIR /src
-    COPY ["DaprFrontEnd/DaprFrontEnd.csproj", "DaprFrontEnd/"]
-    RUN dotnet restore "DaprFrontEnd/DaprFrontEnd.csproj"
+    COPY ["MyFrontEnd/MyFrontEnd.csproj", "MyFrontEnd/"]
+    RUN dotnet restore "MyFrontEnd/MyFrontEnd.csproj"
     COPY . .
-    WORKDIR "/src/DaprFrontEnd"
-    RUN dotnet build "DaprFrontEnd.csproj" -c Release -o /app/build
+    WORKDIR "/src/MyFrontEnd"
+    RUN dotnet build "MyFrontEnd.csproj" -c Release -o /app/build
 
     FROM build AS publish
-    RUN dotnet publish "DaprFrontEnd.csproj" -c Release -o /app/publish
+    RUN dotnet publish "MyFrontEnd.csproj" -c Release -o /app/publish
 
     FROM base AS final
     WORKDIR /app
     COPY --from=publish /app/publish .
-    ENTRYPOINT ["dotnet", "DaprFrontEnd.dll"]
+    ENTRYPOINT ["dotnet", "MyFrontEnd.dll"]
     ```
 
     The preceding *Dockerfile* sequentially performs the following steps when invoked:
 
-    1. Pulls the `mcr.microsoft.com/dotnet/aspnet:5.0` image and names it `base`.
-    1. Sets the working directory to */app*.
-    1. Exposes port `80` and `443`.
-    1. Pulls the `mcr.microsoft.com/dotnet/sdk:5.0` image and names it `build`.
-    1. Sets the working directory to */src*.
-    1. Copies the _DaprFrontEnd/DaprFrontEnd.csproj_ to a new directory named *DaprFrontEnd/*.
-    1. Calls [`dotnet restore`](../../core/tools/dotnet-restore.md) on the project.
-    1. Copies everything from the root directory into the image's root.
-    1. Sets the working directory to _/src/DaprFrontEnd_.
-    1. Calls [`dotnet build`](../../core/tools/dotnet-build.md) on the project.
+    1. Pulls the `mcr.microsoft.com/dotnet/aspnet:6.0` image and names it `base`.
+    2. Sets the working directory to */app*.
+    3. Exposes port `80` and `443`.
+    4. Pulls the `mcr.microsoft.com/dotnet/sdk:6.0` image and names it `build`.
+    5. Sets the working directory to */src*.
+    6. Copies the _MyFrontEnd/MyFrontEnd.csproj_ to a new directory named *MyFrontEnd/*.
+    7. Calls [`dotnet restore`](../../core/tools/dotnet-restore.md) on the project.
+    8. Copies everything from the root directory into the image's root.
+    9. Sets the working directory to _/src/MyFrontEnd_.
+    10. Calls [`dotnet build`](../../core/tools/dotnet-build.md) on the project.
         - Targeting the **Release** configuration and outputs to */app/build*.
```

Comment:
This bullet doesn't look great in the preview - https://review.docs.microsoft.com/en-us/dotnet/architecture/dapr-for-net-developers/getting-started?branch=pr-en-us-27146#add-container-support.